lib/controllers/messages.ts

```diff
@@ -211,7 +211,10 @@ export function CreateMessageHandler(
     if (auth.groups.has(UserGroup.ApiLimitedMessageWrite)) {
       // user is in limited message creation mode, check whether he's sending
       // the message to an authorized recipient
-      if (!userAttributes.authorizedRecipients.has(fiscalCode)) {
+      if (
+        !userAttributes.service.authorizedRecipients ||
+        userAttributes.service.authorizedRecipients.indexOf(fiscalCode) === -1
```

Comment:
uhm, why? `authorizedRecipients` is a `Set`

Reply:
yes I've changed it because now it's retrieved from the document in cosmosdb (as a plain array). 
of course we can make a Set() from the array but it's probably more expensive than just call indexOf here.

Reply:
I suggest to keep a `Set`, since an `Array` is semantically wrong